client-report/src/util/url.js

```diff
@@ -38,13 +38,19 @@ if (0 === document.domain.indexOf("192.168") || (new RegExp(/\.(ssl|n)ip\.io/)).
   urlPrefix = "http://" + document.location.hostname + ":" + document.location.port + "/";
 }
 
+// Take configuration from the environment if supplied.
+if (typeof process.env.POLIS_URL_PREFIX === "string" && process.env.POLIS_URL_PREFIX.length > 0) {
+  urlPrefix = `http://${process.env.PRIMARY_POLIS_URL}/`;
```

Comment:
Was this intended, variable name-wise? Not sure if using two was intended

```suggestion
  urlPrefix = `http://${process.env.POLIS_URL_PREFIX}/`;
```

Reply:
I was going to mention this; Let's minimize the set of variables we need to specify, if at all possible.

The other thing I wanted to point out is that defaulting to `http` is probably not the right way to go.

Given both of these points, should we be using the `polis.config.js` value for `SERVICE_URL` instead, since this would already have `http` or `https` baked in?